src/layout/Content/index.tsx

```diff
@@ -122,6 +128,62 @@ export default function Content() {
       <Route path="/addons/:addonName" component={Addons} />
       <Route path="/addons" component={Addons} />
       <Route path="/users" component={Users} />
+      <Route exact path="/projects" component={Projects} />
+      <Route
+        exact
+        path="/projects/:projectsName"
```

Comment:
Please change all `projectsName` to `projectName`

Reply:
done.